docker-compose.yml

```
@@ -0,0 +1,85 @@
+      MYSQL_BACKUP_PASSWORD: ${MYSQL_BACKUP_PASSWORD}
+      
+      
+    volumes:
+      
+      - db_data:/var/lib/mysql
+      
+      - ./docker/db/init.sh:/docker-entrypoint-initdb.d/init.sh
+      - ./database/schemas:/docker-entrypoint-initdb.d/schemas:ro 
+      - ./database/seeders:/docker-entrypoint-initdb.d/seeders:ro 
+    ports:
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)